src/business/BookService.java

```diff
@@ -0,0 +1,29 @@
+package business;
+
+import dataaccess.BookRepo;
+import dataaccess.BookRepoAccess;
+
+public class BookService {
+	BookRepoAccess bookRepo = new BookRepo();
+	boolean createBookCopy(Book book) throws LibrarySystemException {
+		// validation field for Book form
+		 if (book == null)
+			 throw new LibrarySystemException();
```

Comment:
better to return false here than throw an exception... make sure to adjust the method signature afterwards